tools/pagerduty-migrator/scripts/add_users_pagerduty_to_grafana.py

```diff
@@ -0,0 +1,44 @@
+import os
+import secrets
+import sys
+import requests
+
+from urllib.parse import urljoin
+from pdpyras import APISession
+
+PAGERDUTY_API_TOKEN = os.environ["PAGERDUTY_API_TOKEN"]
+PATH_USERS_GRAFANA = "/api/admin/users"
+URL_GRAFANA = os.environ["URL_GRAFANA"] # Example: http://localhost:3000
```

Comment:
Please consider switching the order of words here (`URL_GRAFANA` -> `GRAFANA_URL`, `USERNAME_GRAFANA` -> `GRAFANA_USERNAME`, `PASSWORD_GRAFANA` -> `GRAFANA_PASSWORD`) since we use the same order of words for `PAGERDUTY_API_TOKEN `, `ONCALL_API_TOKEN`, etc.

Reply:
> Please consider switching the order of words here (`URL_GRAFANA` -> `GRAFANA_URL`, `USERNAME_GRAFANA` -> `GRAFANA_USERNAME`, `PASSWORD_GRAFANA` -> `GRAFANA_PASSWORD`) since we use the same order of words for `PAGERDUTY_API_TOKEN `, `ONCALL_API_TOKEN`, etc.

In this commit, I changed the function name to follow the default.
https://github.com/grafana/oncall/pull/403/commits/a920062c8faea39f95348d131b5b016a8dff62ab